pydantic/networks.py

```diff
@@ -9,7 +9,7 @@
 from ipaddress import IPv4Address, IPv4Interface, IPv4Network, IPv6Address, IPv6Interface, IPv6Network
 from typing import TYPE_CHECKING, Any, ClassVar
 
-from pydantic_core import MultiHostUrl, PydanticCustomError, Url, core_schema
+from pydantic_core import MultiHostHost, MultiHostUrl, PydanticCustomError, Url, core_schema
```

Comment:
Since this module is public, I think should do `Url as _CoreUrl`, same for `MultiHostUrl`.